src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -297,14 +302,13 @@ public Response storePolicy(@QueryParam ("alsoStore") boolean alsoStore, @Valid
     // Basic validation was successful, so try to persist.
     // This may still fail du to unique name violation, so
     // we need to check for that.
-    UUID id;
+    UUID id = null;
     try {
-      id = policy.store(user.getAccount(), policy);
-      // We persisted locally, now tell the engine
       if (!skipEngineCall) {
         FullTrigger trigger = new FullTrigger(policy);
         try {
           engine.storeTrigger(trigger, false, user.getAccount());
+          id = policy.store(user.getAccount(), policy);
```

Comment:
But this won't store in `skipEngineCall` is `true`.

I personally no longer use it and is not that hard to setup and engine, should we remove that now?

Reply:
yes, I started that in a different branch and you are right. Will do that now